chains/container-chains/runtime-templates/frontier/src/xcm_config.rs

```diff
@@ -214,19 +217,42 @@ pub type XcmRouter = WithUniqueTopic<(
     >,
 )>;
 
+/// Filter to ensure an ETH asset is coming from a trusted Ethereum location.
+pub struct EthereumAssetReserve<EthereumLocation, EthereumNetwork>(
```

Comment:
We need to merge this into the single file as soon as another similar PR is merged: https://github.com/moondance-labs/tanssi/pull/1276/files#diff-1b4e48db0caca1a736fa7c2b96ad9ec3e18d907dfa63660414206fae29d5deb4R82